tests/test_mirror.py

```diff
@@ -873,3 +889,16 @@ def test_AclBindMirror(self, dvs, testlog):
         self.remove_neighbor("Ethernet32", "20.0.0.1")
         self.remove_ip_address("Ethernet32", "20.0.0.0/31")
         self.set_interface_status(dvs, "Ethernet32", "down")
+
+    def test_AclBindMirror(self, dvs, testlog):
+        self.setup_db(dvs)
+
+        self._test_AclBindMirror(dvs, testlog)
+        self._test_AclBindMirror(dvs, testlog, create_seq_test=True)
+
+
+# Add Dummy always-pass test at end as workaroud
+# for issue when Flaky fail on final test it invokes module tear-down before retrying
+def test_nonflaky_dummy():
+    pass
```

Comment:
This section is not needed for 201911 vs test. But it does not matter if vs test is not run for 201911.

Reply:
Thanks. Why didn't we need this patch for 201911?